### PR TITLE
*: add HIGH_PRIORITY for the bootstrap SQLs.

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Some important SQLs could failed when TiDB is starting. This PR try to improve this.

### What is changed and how it works?
Add HIGH_PRIORITY for the bootstrap SQLs.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. change the log level to `debug`, add os.Exit(0) after https://github.com/pingcap/tidb/blob/1a9715290b02f07fe98851b2fed7a2accb18113c/server/server.go#L241
1. start a cluster with 2 TiDB servers.
1. check the SQLs in the two log files.

Code changes
- no

Side effects

 - Possible performance regression

Related changes

- no

PTAL @winkyao @breeswish

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -36,11 +36,11 @@ import (
 	"math/rand"
 	"net"
 	"net/http"
+	// For pprof
```
Comment (winoros) [inline server/server.go]: This change is made by `gofmt`?

Comment (jackysp) [reply server/server.go]: Yes.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**